lldb/test/API/tools/lldb-dap/databreakpoint/TestDAP_setDataBreakpoints.py

```diff
@@ -171,3 +171,35 @@ def test_functionality(self):
         self.continue_to_next_stop()
         x_val = self.dap_server.get_local_variable_value("x")
         self.assertEqual(x_val, "10")
+
+    @skipIfWindows
+    def test_breakpoint_reason(self):
+        """Tests setting data breakpoints on variable. Verify that the breakpoint has the correct reason and description in the stopped event."""
```

Comment:
I would break the description into multiple lines

Reply:
done